### PR TITLE
feat(product-page): hydrate template with real data

### PR DESCRIPTION
**Pull request by mkucmus**

### CODE REVIEW

File: test-page/src/views/Category.vue [hunk 1]
```diff
@@ -125,7 +125,7 @@
             "
             :regular-price="product.calculatedPrice.unitPrice"
             :isOnWishlist="false"
-            @click:wishlist="toggleWishlist(i)"
+            @click:wishlist="$router.push({name: 'product', params: {id: product.id}})"
```
Comment (mkucmus) [inline test-page/src/views/Category.vue]: temporary workaround, just ignore it

Comment (patzick) [reply test-page/src/views/Category.vue]: we don't want to change route on withlist button click :P

File: test-page/src/views/Product.vue [hunk 2]
```diff
@@ -451,9 +390,145 @@ export default {
...
+    },
+    sizes() {
+      if (!this.product && !this.product.children) {
+        return []
+      }
+
+      return this.extractOptions("size")
+    },
+    reviews() {
+      return this.product.productReviews
```
Comment (patzick) [inline test-page/src/views/Product.vue]: you're checking `reviews.length` much, so better ensure it always is at least an empty array
```suggestion
      return this.product.productReviews || []
```

File: test-page/src/views/Category.vue (hunk 1 above)
Comment (mkucmus) [reply test-page/src/views/Category.vue]: It's fine, I restored it

**Pull request merged**